server/ControlPlane/Buffers/AzureBlobBufferProvider.cs

```diff
@@ -84,6 +84,27 @@ public async Task<Buffer> CreateBuffer(Buffer buffer, CancellationToken cancella
         return await _repository.CreateBuffer(buffer, storageAccountId, cancellationToken);
     }
 
+    public async Task<IList<string>> DeleteBuffers(IList<string> ids, CancellationToken cancellationToken)
+    {
+        var storageAccountIds = await _repository.GetBufferStorageAccountIds(ids, cancellationToken);
+        var deletedIds = new List<string>();
+        foreach (var id in ids)
+        {
+            var storageAccountId = storageAccountIds.First(sa => sa.bufferId == id).accountId;
+            if (!storageAccountId.HasValue)
+            {
+                continue;
+            }
+
+            var refreshingClient = await GetRefreshingServiceClient(storageAccountId.Value, cancellationToken);
+            var containerClient = refreshingClient.ServiceClient.GetBlobContainerClient(id);
+            await containerClient.DeleteIfExistsAsync(cancellationToken: cancellationToken);
```

Comment:
Since we are able to push Tyger to ~300K runs/hour, where there are two buffers created per run, it's worth considering what would happen if a TTL were enabled on all of them. Would we be able to keep up here?